src/preact/compat.js

```diff
@@ -1,4 +1,4 @@
-import {options} from /*OK*/ 'preact';
+import {options, toChildArray} from /*OK*/ 'preact';
```

Comment:
nit: could we just import children from preact/compat instead of re-implementing?

Reply:
That would bring back all the side-effects we removed in https://github.com/ampproject/amphtml/pull/36976